src/core_plugins/kibana/public/visualize/editor/sidebar.html

```diff
@@ -14,10 +14,14 @@
         <!-- tabs -->
         <ul class="nav navbar-nav">
           <li ng-class="{active: sidebar.section == 'data'}" ng-show="sidebar.showData">
-            <a class="navbar-link active" ng-click="sidebar.section='data'">Data</a>
+            <a class="vis-editor-subnav-link"
+               ng-class="{'is-vis-editor-sub-nav-link-selected': sidebar.section == 'data'}"
+               ng-click="sidebar.section='data'">Data</a>
```

Comment:
Small nit, the attributes should all be on their own lines, the `>` of the opening tag should be on its own line, and the text and closing tag should be on their own lines:

```html
<a
  class="vis-editor-subnav-link"
  ng-class="{'is-vis-editor-sub-nav-link-selected': sidebar.section == 'data'}"
  ng-click="sidebar.section='data'"
>
  Data
</a>
```